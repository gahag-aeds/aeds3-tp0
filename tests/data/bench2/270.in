270 135
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 
Add 1 90
Max 125 214
Sub 56 145
Sum 45 134
Sub 114 203
Min 115 204
Add 85 174
Max 71 160
Sub 180 269
Min 25 114
Sub 42 131
Sum 126 215
Sub 82 171
Min 61 150
Add 83 172
Max 80 169
Sub 18 107
Min 133 222
Sub 74 163
Min 163 252
Sub 90 179
Min 127 216
Add 65 154
Sum 30 119
Sub 60 149
Sum 60 149
Add 149 238
Min 175 264
Add 29 118
Min 1 90
Sub 146 235
Sum 30 119
Add 125 214
Sum 21 110
Sub 126 215
Max 59 148
Add 7 96
Min 31 120
Add 1 90
Max 59 148
Add 55 144
Sum 42 131
Sub 4 93
Max 8 97
Sub 154 243
Max 86 175
Sub 88 177
Min 172 261
Add 91 180
Sum 33 122
Sub 154 243
Sum 180 269
Add 159 248
Sum 90 179
Add 81 170
Sum 90 179
Add 149 238
Sum 102 191
Sub 84 173
Min 49 138
Sub 102 191
Min 49 138
Sub 130 219
Max 47 136
Add 121 210
Min 76 165
Add 157 246
Min 127 216
Sub 158 247
Max 29 118
Add 5 94
Sum 84 173
Add 71 160
Min 61 150
Sub 92 181
Sum 96 185
Sub 18 107
Max 179 268
Add 87 176
Min 160 249
Add 83 172
Min 112 201
Add 159 248
Min 61 150
Add 73 162
Min 112 201
Sub 150 239
Sum 93 182
Add 33 122
Max 53 142
Sub 142 231
Max 134 223
Add 153 242
Min 91 180
Sub 52 141
Sum 93 182
Sub 38 127
Max 80 169
Add 91 180
Sum 15 104
Add 109 198
Min 148 237
Add 151 240
Sum 51 140
Sub 28 117
Max 62 151
Sub 146 235
Sum 45 134
Sub 112 201
Max 53 142
Add 25 114
Max 14 103
Sub 36 125
Min 55 144
Sub 126 215
Min 109 198
Sub 38 127
Max 95 184
Add 21 110
Min 70 159
Add 19 108
Min 34 123
Sub 76 165
Sum 171 260
Add 177 266
Max 179 268
Add 135 224
Min 34 123
Add 79 168
Min 97 186
Add 101 190
Max 59 148
Sub 64 153
Sum 123 212
Add 109 198
