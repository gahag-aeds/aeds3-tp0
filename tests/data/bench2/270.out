99
4316
-1
99
1
4511
-1
99
-2
-2
-2
4171
3884
-1
-1
4200
4210
97
0
99
4315
101
97
-3
4372
4581
4211
4292
4171
0
0
99
-3
-2
99
4174
2
4133
97
-4
-4
1
-3
4279
102
97
-3
4249
98
4537
-3
4259
99
4342
99
102
-1
-5
98
-1
0
4782
98
0
-3
100
4493
